code/controllers/subsystems/machinery.dm

```diff
@@ -69,13 +69,13 @@
 
 		var/start_tick = world.time
 
-		if (M.isprocessing)
+		if (M.machinery_processing)
 			processes_this_tick++
 			switch (M.process())
```

Comment:
I have a crazy idea. But.

Uh.

Why not machinery_process()?

Then you could. Like. Have a machine run two separate stacks. One being lighter and in the general processor. Or no bueno? Without it, this change is kinda worthlesssss.